src/test/regress/expected/citus_local_table_triggers.out

```diff
@@ -33,18 +33,18 @@ NOTICE:  executing the command locally: SELECT worker_apply_inter_shard_ddl_comm
 ---------------------------------------------------------------------
 BEGIN;
     CREATE TABLE distributed_table(value int);
-    SELECT create_distributed_table('distributed_table', 'value');
- create_distributed_table
----------------------------------------------------------------------
-
-(1 row)
-
     CREATE FUNCTION insert_42() RETURNS trigger AS $insert_42$
     BEGIN
         INSERT INTO distributed_table VALUES (42);
         RETURN NEW;
     END;
     $insert_42$ LANGUAGE plpgsql;
+    SELECT create_distributed_table('distributed_table', 'value');
```

Comment:
We changed the orders because create_distributed_table happens in parallel mode, and `CREATE FUNCTION` fails otherwise